bindings/python/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Apache License 2.0
 
-open-space-toolkit-physics~=1.1.1
+open-space-toolkit-physics~=1.1
```

Comment:
Is there a reason you're pinning to the major version? I think we had decided that we will pin to the minor version no?

Reply:
+1 I believe this will allow Python to automatically update minor versions, which is different than how cpp dependencies get propagated. This caused some syncing issues between cpp/Python in the past.

Reply:
Reverted!